koku/api/resource_types/aws_ec2_compute_os/view.py

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright 2024 Red Hat Inc.
+# SPDX-License-Identifier: Apache-2.0
+#
+"""View for AWS accounts."""
```

Comment:
Same here os I guess?

Reply:
yes, good catch 👀 . I've updated the descriptions